.github/actions/setup-postgres-windows/setup_db.sh

```diff
@@ -0,0 +1 @@
+../../../test/setup_db.sh
```

Comment:
Since this is the same in a few places should we just make it one action under `setup-postgres-all`? I'm not opposed to this either though if it feels better organized to you.

Reply:
I was a little limited in what I could do here. I wasn't able to figure out how to run conditional steps in the same actions. I attempted to make a single action, and only run the step for the particular OS it was targeting. For some reason, I couldn't use something like: `if: os == 'windows'`. I landed on splitting up the actions per OS and moving the conditional logic in the workflow https://github.com/dbt-labs/dbt/blob/bdc42e3d040e68122ab5a8b643f87597f459d69a/.github/workflows/integration.yml#L50-L66

I don't think I had to symlink this file though! I think I could just de `bash test/setup_db.sh` in the action, but I wasn't 100%. I'll try this out, but I won't consider it a blocker for merging.

Reply:
Fair enough. It's not actually that awkward.